content/en/platform/corda/4.11/enterprise/legal-info/legal-info-4.11.html

```diff
@@ -1,10 +1,13 @@
 ---
 title: Corda Enterprise Edition 4.11 third-party software notices and licence information
 date: '2023-11-24'
+section_menu: corda-enterprise-4-11
+project: corda
+version: 'Enterprise 4.11'
 menu:
   corda-enterprise-4-11:
-    identifier: corda-enterprise-4-11-legal-info
-    parent: corda-enterprise-4-11-release-notes
+    identifier: corda-enterprise-4-11-0-legal-info
+    parent: corda-enterprise-4-11-legal-info
     weight: 400
     name: Third-party software licence information - 4.11
```

Comment:
```suggestion
    name: 4.11
```
Think that's what you did in the other versions?